mono/mini/mini-exceptions.c

```diff
@@ -712,7 +712,7 @@ unwinder_unwind_frame (Unwinder *unwinder,
 
 		/* Process debugger invokes */
 		/* The DEBUGGER_INVOKE should be returned before the first interpreter frame for the invoke */
-		if (unwinder->last_frame_addr > (gpointer)(*lmf)) {
+		if (unwinder->last_frame_addr < (gpointer)(*lmf)) {
```

Comment:
Doesn't that depend on the orientation of the stack? We do support platforms that have different stack orientation, so maybe having the same approach would be useful.

Reply:
what platforms are those?

Reply:
Turns out I was mistaken. I remembered some places in the code checked for it, but that wasn't enough to support it fully.